tests/suite/test_smoke.py

```diff
@@ -73,6 +73,7 @@ def fin():
 
 @pytest.mark.smoke
 @pytest.mark.ingresses
+@pytest.mark.ingresses_smoke
```

Comment:
this is just 1 test so we can keep it as part of `ingresses` marker

Reply:
@vepatel it seems to be one of the longest parts of the `ingresses` marker, but now that I see some more runs it does look like it makes sense to have it part of the other group, will get it changed!